src/_P092_DLbus.ino

```diff
@@ -676,7 +695,7 @@ void Plugin_092_SetIndices(int DeviceIndex) {
   P092_DataSettings.IdxOutput     = P092_DataSettings.IdxExtSensor + 2 * P092_DataSettings.MaxExtSensors;
   P092_DataSettings.IdxDrehzahl   = P092_DataSettings.IdxOutput + P092_DataSettings.OutputBytes;
   P092_DataSettings.IdxAnalog     = P092_DataSettings.IdxDrehzahl + P092_DataSettings.SpeedBytes;
-  P092_DataSettings.IdxHmRegister = P092_DataSettings.IdxAnalog + P092_DataSettings.AnalogBytes;
+  P092_DataSettings.IdxHmRegister = P092_DataSettings.IdxAnalog + P092_DataSettings.AnalogBytes * P092_DataSettings.MaxAnalogOuts;
```

Comment:
Is this valid for `MaxAnalogOuts == 0` ?

Reply:
Hi TD-er,
thank you for your review. good point but i think it behaves correctly...

there are devices where there is no Analog output (i.e. UVR1611). in this case the heatmeter register is placed directly after the "drehzahl" byte (see the attached screenshot from the UVR1611 protocol). 

![grafik](https://user-images.githubusercontent.com/73192191/97277829-4254fa80-1839-11eb-88ba-fc441a227526.png)

i.e. for device 1611 the MaxAnalogOuts is set to 0. in this case IdxHmRegister becomes IdxAnalog, which is the next byte after the "Drehzahlstufe" byte.
the only thing is, that IdxAnalog is not correct in this case (because no analog output exists in this case) but since Analog output is not choosable in the webform for this device, i don't think this is a problem.

for clarification i also add the according part of the UVR61-3 protocol.

![grafik](https://user-images.githubusercontent.com/73192191/97278151-b099bd00-1839-11eb-976b-9ede51729807.png)

greets peter

Reply:
Well I wondered since the IdxAnalog is used to compute the next register address.
But as long as the order of registers remains without a gap, then it is indeed no problem.

Reply:
Double checked also the protocols of the other devices... no gaps